pkg/agent/flowexporter/connections/deny_connections_test.go

```diff
@@ -74,6 +74,7 @@ func TestDenyConnectionStore_AddOrUpdateConn(t *testing.T) {
 	denyConnStore.AddOrUpdateConn(&testFlow, refTime.Add(-(time.Second * 20)), uint64(60))
 	expConn := testFlow
 	expConn.DestinationServicePortName = servicePortName.String()
+	expConn.LastExportTime = refTime.Add(-(time.Second * 20))
```

Comment:
I think `denyConnStore.AddOrUpdateConn` has already taken care of assigning value to `LastExportTime`?